lib/recase.dart

```diff
@@ -106,6 +106,65 @@ class ReCase {
     return words.join(separator);
   }
 
+  // following APA Style guide:
+  // - https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case
+  String _getTitleCase({String separator = ' '}) {
+    // reset all to lowercase
+    List<String> words = this._words.map((word) => word.toLowerCase()).toList();
+    // exclusion list
+    List<String> exclusionListEng = [
+      // articles
+      'a',
+      'an',
+      'the',
+      // coordinating conjunctions
+      'and',
+      'but',
+      'for',
+      // 'nor', // not NYT-style
+      'or',
+      // 'so', // not NYT-style
+      'yet',
+      // short prepositions (< 4 chars)
+      'as',
+      'at',
+      'by',
+      'en',
+      'if',
+      'in',
+      'of',
+      // 'off', // not NYT-style
+      'on',
+      'per',
+      'to',
+      // 'up', // not NYT-style
+      'v.',
+      'via',
+      'vs.'
+    ];
+    List<String> punctuationList = [':', '--', '.', '?', '!'];
+
+    // apply Title Case
+    words = words.asMap().entries.map((word) {
+      var isFirstWord = word.key == 0;
+      var isAfterPunctuation =
+          !isFirstWord && punctuationList.indexOf(words[word.key - 1]) > 0;
+      var isExcluded = exclusionListEng.indexOf(word.value) > 0;
```

Comment:
```suggestion
          !isFirstWord && punctuationList.contains(words[word.key - 1]);
      var isExcluded = exclusionListEng.contains(word.value);
```
I think you should use `contains`, because `indexOf(...) > 0` will return false if the value is the first element in the list, but it should be true for that case.

Reply:
done 👍🏼